frontend_build/src/htmlhint_custom.js

```diff
@@ -99,38 +107,42 @@ HTMLHint.addRule({
               reporter.error('Top-level content should be indented two spaces.', event.line, event.col, self, event.raw);
             }
             if (match && match[3].length !== 2) {
-              var offset = (event.raw.match(/\n/g) || []).length;
+              var offset = (eventData.match(/\n/g) || []).length;
               reporter.error('Top-level content should be surrounded by one empty line.', event.line+offset, 1, self, event.raw);
             }
           }
         }
       }
     });
     parser.addListener('tagstart', function(event) {
-      if (!stack.length) {
+      var eventData = clean(event.raw);
+      if (!stack.length && event.lastEvent) {
         if (event.lastEvent.type === "start") {
           if (event.line !== 1) {
             reporter.error('Content should start on the first line of the file.', event.line, event.col, self, event.raw);
           }
         }
-        else if (event.lastEvent.raw !== "\n\n\n") {
-          reporter.error('Need two endlines between top-level tags.', event.line, event.col, self, event.raw);
+        else if (event.lastEvent.raw && clean(event.lastEvent.raw) !== "\n\n\n") {
+          reporter.error('Need two endlines between top-level tags.', event.line, event.col, self, event.raw); //git gutter
```

Comment:
hey Jessica - want to just take out this note-to-self? :)

this is the plugin btw: https://packagecontrol.io/packages/GitGutter